src/Microsoft.PowerApps.TestEngine/SingleTestRunner.cs

```diff
@@ -205,7 +208,7 @@ public async Task RunTestAsync(string testRunId, string testRunDirectory, TestSu
                                 }
                             }
 
-                            var message = $"{{ \"TestName\": {testCase.TestCaseName}, \"BrowserConfiguration\": {JsonConvert.SerializeObject(browserConfig)}}}";
+                            var message = $"{{ \"BrowserConfiguration\": {JsonConvert.SerializeObject(browserConfig)}}}";
```

Comment:
double curly brackets? would end up looking like the below, and basically would look like array of arrays with one element for the user - 
```json
{
  { 
    "BrowserConfiguration": { 
        "Browser" : "Chromium"
    }
  }
}
```

I think just one set is enough.. Removing the extra set of curly brackets would then look like -  
```json
{
  "BrowserConfiguration": {
    "Browser": "Chromium",
  }
}
```

Reply:
This is intentional syntax and is already in the second format.

Reply:
That's correct, my bad!